README.md

```diff
@@ -0,0 +1,68 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Pair-Portfolio
+
+> One paragraph statement about the project.
+
+![screenshot](media/hello-microverse.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
please add the major language, framework and technologies used, if any :grinning: 
It makes your work feel professional when yous state the built with details